src/translations/en.json

```diff
@@ -3248,9 +3248,9 @@
                   "has_optional_response": "This service can return a response, if you want to use the response, enter the name of a variable the response will be saved in",
                   "has_response": "This service returns a response, enter the name of a variable the response will be saved in",
                   "description": {
-                    "service_based_on_template": "Call a service based on a template on {targets}",
-                    "service_based_on_name": "Call a service ''{name}'' on {targets}",
-                    "service_name": "{domain} ''{name}'' on {targets}",
+                    "service_based_on_template": "Call a service based on a template{hasTargets, select, \n true { on {targets}} \n other {} \n }",
+                    "service_based_on_name": "Call a service ''{name}''{hasTargets, select, \n true { on {targets}} \n other {} \n }",
+                    "service_name": "{domain} ''{name}''{hasTargets, select, \n true { on {targets}} \n other {} \n }",
```

Comment:
Should we split each translation in two keys to simplify translator work?

Reply:
We've already got dozens of keys using a similar pattern. Should I consider `{select}` as always discouraged? Having two translations for the same sentence seems cluttery, and this does not seem exceptionally complex, but I will follow guidance if we have some.

Reply:
I've introduced many of these, and the general feedback from those pull requests was to split the sentence when there were multiple selects, nested selects or it became pretty long in number of characters. 
If it's really needed, I would suggest to introduce keys for when no target exists, so the existing translations don't need to be translated.

Reply:
Yes, agreed, it saves missing translations (old ones would need to get updated) and makes it a bit easier for translators (although I'm surprised how well it goes actually!)